.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode
```

Comment:
This is fine, but you may want to consider adding `.vscode` to your [global gitignore](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)